Tensile/KernelWriterAssembly.py

```diff
@@ -250,6 +250,11 @@ def getCompileArgs(self, sourceFileName, objectFileName, *moreArgs, isa=None, wa
   def getLinkCodeObjectArgs(self, objectFileNames, coFileName, *moreArgs):
     rv = [globalParameters['AssemblerPath'],
           '-target', 'amdgcn-amd-amdhsa']
+    # Add build-id for builds with rocm 5.3+
+    compilerVer = globalParameters['HipClangVersion'].split(".")[:2]
+    compilerVer = [int(c) for c in compilerVer]
+    if len(compilerVer) >= 2 and (compilerVer[0] > 5 or (compilerVer[0] == 5 and compilerVer[1] > 2)):
+      rv += ["-Xlinker", "--build-id"]
```

Comment:
Do we need to specify this for os.name=="nt" case?
At line 120 of TensileCreateLibrary.py, parameters are set without using this function.
Should we add --build-id option separately?

Reply:
Updated the nt code to also call this function